### PR TITLE
Add OpInfo tests for torch.baddbmm

### PR DESCRIPTION
**Pull request by anjali411**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#55566 Add OpInfo tests for torch.baddbmm**

### CODE REVIEW

File: torch/testing/_internal/common_methods_invocations.py [hunk 1]
```diff
@@ -4415,14 +4449,6 @@ def method_tests():
...
         ('addbmm', (), ((S, S, S), (S, S, M)), 'scalar_broadcast_lhs_coef', (), (), (), ident,
          {'beta': 0.2, 'alpha': 0.6}),
-        ('baddbmm', (S, S, M), ((S, S, S), (S, S, M)),),
-        ('baddbmm', (1,), ((S, S, S), (S, S, M)), 'broadcast_lhs'),
-        ('baddbmm', (S, S, M), ((S, S, S), (S, S, M)), 'coef', (), (), (), ident, {'beta': 0.2, 'alpha': 0.6}),
-        ('baddbmm', (1,), ((S, S, S), (S, S, M)), 'broadcast_lhs_coef', (),
-         (), (), ident, {'beta': 0.2, 'alpha': 0.6}),
-        ('baddbmm', (), ((S, S, S), (S, S, M)), 'scalar_broadcast_lhs'),
-        ('baddbmm', (), ((S, S, S), (S, S, M)), 'scalar_broadcast_lhs_coef', (), (), (), ident,
-         {'beta': 0.2, 'alpha': 0.6}),
```
Comment (imaginary-person) [inline torch/testing/_internal/common_methods_invocations.py:4425]: This one was removed by mistake

**Comment by imaginary-person** [comment]
BTW, I noticed that this PR doesn't have the `hackathon` label. Can I get started on an `OpInfo` that hasn't been ported yet? Thanks!

**Comment by anjali411** [comment]
> BTW, I noticed that this PR doesn't have the `hackathon` label. Can I get started on an `OpInfo` that hasn't been ported yet? Thanks!

Hi @imaginary-person a bunch of PRs were created to port method_tests today. No one worked on the reshaping ops so you could get started with that in the time @mruberry updates the tracker.

**Comment by imaginary-person** [comment]
> No one worked on the reshaping ops so you could get started with that

Thanks, @anjali411, but if your hackathon is still going on, then I can certainly wait. :)

**Comment by mruberry** [comment]
> > No one worked on the reshaping ops so you could get started with that
> 
> Thanks, @anjali411, but if your hackathon is still going on, then I can certainly wait. :)

Tracker should be updated now if you're interested in porting more ops. I'm not sure how easy/hard porting shape ops would be.

**Comment by anjali411** [comment]
closing this in favor of https://github.com/pytorch/pytorch/pull/56502

**Pull request closed without merging**